### PR TITLE
Multiplex readme text

### LINKED ISSUE #244: Add multiplexed samples README template
Add a new README template that will be used for multiplexed samples downloads.

### PR DESCRIPTION
**Pull request by jashapiro**
## Issue Number

closes #244

## Purpose/Implementation Notes

Updates readme file for multiplexed data with appropriate info.

I mostly followed the main readme file text, with additions for the multiplexed data file structure and expectations, with links to the appropriate sections of the docs.

## Types of changes

All updates to template markdown file.

- Bugfix (non-breaking change which fixes an issue)
(documentation update)

## Functional tests

I have not tested variable substitution, though these are unchanged from `readme.md`

## Checklist

- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: api/scpca_portal/templates/readme_multiplexed.md [hunk 1]
```diff
@@ -4,24 +4,34 @@ The [Single-cell Pediatric Cancer Atlas](https://scpca.alexslemonade.org) is a d
 
 ## Contents
 
-This download includes gene expression data from libraries processed using spatial transcriptomics and associated metadata for samples from project [{project_accession}]({project_url}) in the ScPCA portal.
+This download includes single-cell or single-nuclei gene expression files and associated metadata for samples from project [{project_accession}]({project_url}) in the ScPCA portal.
 
-For all spatial transcriptomics libraries (indicated by the `SCPCL` prefix) , a `SCPCL000000_spatial` folder will be nested inside the corresponding sample folder (`SCPCS` prefix) in the download.
-See the [FAQ section about samples and libraries](https://scpca.readthedocs.io/en/latest/faq.html#what-is-the-difference-between-samples-and-libraries) for more information.
+This project includes multiplexed samples, where multiple biological samples have been combined into libraries using cellhashing or similar technologies.
+The data files are divided into folders named by the set of samples that the libraries within contain (an underscore-separated list of sample ids, each with an `SCPCS` prefix).
```
Comment (allyhawkins) [inline api/scpca_portal/templates/readme_multiplexed.md:10]:
```suggestion
Each folder will be named with an underscore-separated list of sample ids (containing the `SCPCS` prefix) that correspond to the samples that are multiplexed and combined to create a given library(s). 
Within each folder are the gene expression data for each library containing the samples listed in the folder name.
```
I don't know if this is much better, but something about this sentence was confusing so I tried to clarify a bit?

File: api/scpca_portal/templates/readme_multiplexed.md [hunk 2]
```diff
@@ -4,24 +4,34 @@ The [Single-cell Pediatric Cancer Atlas](https://scpca.alexslemonade.org) is a d
...
+The `bulk_metadata.tsv` file contains associated metadata for all samples with bulk RNA-seq data.
 
 See the [Downloadable files](https://scpca.readthedocs.io/en/latest/download_files.html) section in our documentation for more detailed information on files included in the download.
 
-For more information on how the spatial libraries were processed, see the [Spatial Transcriptomics section in the Processing information](https://scpca.readthedocs.io/en/latest/processing_information.html#spatial-transcriptomics) page of the ScPCA Portal documentation.
+## Usage
+
+For instructions on using the RDS files, please see [FAQ: How do I use the provided files in R?](https://scpca.readthedocs.io/en/latest/faq.html#how-do-i-use-the-provided-rds-files-in-r) or [FAQ: What if I want to use Python instead of R?](https://scpca.readthedocs.io/en/latest/faq.html#what-if-i-want-to-use-python-instead-of-r)
+
+Note that
```
Comment (allyhawkins) [inline api/scpca_portal/templates/readme_multiplexed.md:34]:
```suggestion
```
I think this was here by mistake.

**Review by allyhawkins — COMMENTED** [review]
This looks good, the only major comment I have is about adding the link to the getting started with multiplex.

File: api/scpca_portal/templates/readme_multiplexed.md (hunk 2 above)
Comment (jashapiro) [reply api/scpca_portal/templates/readme_multiplexed.md]: I think this was actually where I meant to add a link to the multiplex getting started section...

File: api/scpca_portal/templates/readme_multiplexed.md (hunk 1 above)
Comment (jashapiro) [reply api/scpca_portal/templates/readme_multiplexed.md]: I really struggled with this sentence, and I didn't love where it ended up either. I tried again with the latest revision; let me know what you think!

**Review by allyhawkins — APPROVED** [review]
LGTM!

**Pull request merged**